src/Components/__tests__/Footer.spec.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import {shallow} from 'enzyme';
-import Footer from '../Components/Footer'
+import Footer from '../Footer'
 
 describe("Footer", () => {
-
-    const footer = shallow(<Footer />);
+    let footer;
+    beforeEach(
```

Comment:
This is definitely a nit pick on my part, so feel free to veto haha! What do you think of structuring the `beforeEach`es with this formatting: 
```
beforeEach(() => {
        footer = shallow(<Footer />)
})
```
So that it's similar to how we structure our `it` statements

Reply:
Oh yes. Thanks that makes sense.